pkg/runner/job_runner.go

```diff
@@ -374,6 +354,65 @@ func (jr *JobRunner) Run(ctx xcontext.Context, j *job.Job, resumeState *job.Paus
 	return nil, nil
 }
 
+func (jr *JobRunner) lockRefresher() {
+	interval := jr.targetLockDuration / 10 * 9
```

Comment:
There was a comment here before why this is `/ 10 * 9` , can we keep that?

Reply:
done